private/weather/package.json

```diff
@@ -10,7 +10,7 @@
     "build:types": "tsc -p tsconfig.types.json",
     "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
     "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
-    "test": "vitest run"
+    "test": "vitest run --passWithNoTests"
```

Comment:
This is code generated from https://github.com/awslabs/smithy-typescript/pull/1065